learning/dashboard/group9/readme.md

```diff
@@ -0,0 +1,105 @@
+# How to install and setup NodeJS and npm?
+- **step 1:**
+Open the Terminal on your Ubuntu system. You can do this by pressing the **"Ctrl+Alt+T"** keys simultaneously.
+- **step 2:**
+Update your package list by typing the following command in the Terminal:
+
+   `sudo apt-get update`
+ - **step3:**
+  Install Nodejs using the following command: 
+  
+    `sudo apt get install node js`
```

Comment:
Is this correct? I don't think this will work.  is the package `node js` ?

Reply:
no, there is no space in between
we will do the required changes